hoomd/operations.py

```diff
@@ -230,12 +230,12 @@ def __contains__(self, operation):
     def __iter__(self):
         """Iterates through all contained operations."""
         integrator = (self._integrator,) if self._integrator else []
-        yield from chain(self._tuners, self._updaters, integrator,
-                         self._writers, self._computes)
```

Comment:
Is this change important to fixing the issue? I liked the previous ordering as it follows the same order the operations are applied during `run` (except computes which may be triggered at an time during the step)

Reply:
No it isn't, and that is good reason to revert.